lib/datasource/docker/common.spec.ts

```diff
@@ -70,4 +71,51 @@ describe(getName(), () => {
       `);
     });
   });
+  describe('getAuthHeaders', () => {
+    beforeEach(() => {
+      nock('https://my.local.registry')
```

Comment:
```suggestion
      httpMock.scope('https://my.local.registry')
```

Simply replace `nock(url)` with `httpMock.scope(url)`